.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,74 @@
+Based on the nature of your change, please use the relevant template below and modify to suit your needs.
```

Comment:
I'm not the best sub editor in the world, but it could be something as simple as:

"Styleguide is semantically version. If semver is unfamiliar to you, have a read (of)[link]; otherwise choose the right template for your version bump below"

Don't use that! But it's succinct. Thoughts?